examples/observability/adot-amp-grafana-for-haproxy/main.tf

```diff
@@ -145,73 +145,73 @@ resource "grafana_dashboard" "haproxy_dashboards" {
   config_json = file("${path.module}/dashboards/default.json")
 }
 
-#Configure AWS Managed Prometheus rule groups
 resource "aws_prometheus_rule_group_namespace" "haproxy" {
   name         = "haproxy_rules"
   workspace_id = module.eks_blueprints.amazon_prometheus_workspace_id
-  data         = <<EOF
+
+  data = <<-EOF
```

Comment:
we should use the `<<-EOF` indented version versus the `<<EOF` which means all text should be aligned to far left side of page or else this is shown as whitespace